get.go

```diff
@@ -66,12 +72,13 @@ func init() {
 		Netrc: true,
 	}
 
-	Getters = map[string]Getter{
-		"file":  new(FileGetter),
-		"git":   new(GitGetter),
-		"hg":    new(HgGetter),
-		"http":  httpGetter,
-		"https": httpGetter,
+	Getters = []Getter{
+		new(GitGetter),
+		new(HgGetter),
+		new(SmbClientGetter),
+		new(SmbMountGetter),
```

Comment:
Here they are @azr

Reply:
Yes I was looking for that, thx !

Reply:
Ah btw I think we should add a comment here stating that the order is importer